packages/api-admin-users-cognito/src/createAdminUsers.ts

```diff
@@ -94,34 +97,61 @@ export const createAdminUsers = ({
             };
 
             let result;
-            await this.onUserBeforeCreate.publish({ user, inputData: data });
-            /**
-             * Always delete `password` from the user data!
-             */
-            delete (user as any)["password"];
-            try {
-                result = await storageOperations.createUser({ user });
-            } catch (err) {
-                throw WebinyError.from(err, {
-                    message: "Could not create user.",
-                    code: "CREATE_USER_ERROR",
-                    data: { user: result || user }
+
+            const wcpProjectEnvironment = getWcpProjectEnvironment();
```

Comment:
This wasn't the idea we discussed on the call.

We said we'll load the project environment outside of this file (upon calling `createAdminUsers` method) and we'll send the project environment to this `createAdminUsers` as an arg. This way we're not polluting / introducing a new dependency directly into this method, but we allow for this value to be passed outside of it. 

Let's do it like this.

1. Let's introduce a new `wcp: WcpContextObject` param for `createAdminUsers`
2. On the WcpContextObject side, let's add the incrementSeats / decrementSeats methods, which we'll then call within `createAdminUsers`. Much better than having all this WCP-specific code in a plugin whose purpose is something completely different.